README.md

```diff
@@ -94,14 +94,13 @@ The `mesh` command supports the following flags:
 
 #### Basic default manifest
 
-The following command generates a manifest with the compiled in default profile and charts:
+The following command generates a manifest with the compiled-in profiles and charts:
```

Comment:
compiled-in is ok, but please change back to default profile, because that's what it will use (data/profiles/default.yaml).